cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -228,9 +228,8 @@ def verify_revert_snapshot(full_test_vm, provider, soft_assert, register_event,
         assert ssh_client.run_command('test -e snapshot2.txt') == 1
 
 
-@pytest.mark.meta(
-    blockers=[BZ(1805803, unblock=lambda provider: not provider.one_of(RHEVMProvider),
-                 ignore_bugs={1745065})], automates=[1805803])
+@pytest.mark.rhv1
+@pytest.mark.provider([VMwareProvider])
```

Comment:
I think if we're going to remove RHV from the parametrization here, we should include comments concerning the BZ's, the fact that they aren't getting fixed, etc.

The other option here is to use `pytest.mark.uncollectif`, evaluating the provider type and BZ blocking status to uncollect the test case instead of skipping it.

This will make it so the test case isn't collected and included in reports, when it will not get fixed, instead of just being skipped at runtime.

Reply:
Uncollectif added.